fabric/src/main/java/dev/latvian/kubejs/ui/fabric/FabricActions.java

```diff
@@ -13,7 +13,7 @@
 	Consumer<Screen> FABRIC_MOD_LIST = screen -> {
 		if (Platform.isModLoaded("modmenu"))
 		{
-			mc().setScreen(new ModsScreen(screen));
+			ModMenuApi.createModsScreen(screen);
```

Comment:
I think this is supposed to be mc().setScreen(ModMenuApi.createModsScreen(screen))? It's just creating a screen, but you never set it.

Reply:
...how do I manage to screw up the simplest of things 😅